ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-release-4.12-presubmits.yaml

```diff
@@ -815,7 +960,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp-disruptive,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Also not sure why `make jobs` is now marking all jobs as always run. Wasn't doing that the last time I updated these.
Possibly an update in ci-operator but need to confirm what to tweak in the presubmits to not run these. Don't just want to edit these back manually.